auto_round/special_model_handler.py

```diff
@@ -15,7 +15,7 @@
 import torch
 from collections import UserDict
 shareable_keywords = ("position_ids", "cache_position", "position_embeddings",
-                        "position_embeddings_global", "position_embeddings_local")
+                        "position_embeddings_global", "position_embeddings_local", "slope_rate")
```

Comment:
As we may release new version soon, to minimize potential risks, we  better temporarily relocate the newly introduced keys to the specialized model handler.